config/crd/standard/gateway.networking.k8s.io_gatewayclasses.yaml

```diff
@@ -491,12 +446,7 @@ spec:
                       - Unknown
                       type: string
                     type:
-                      description: |-
-                        type of condition in CamelCase or in foo.example.com/CamelCase.
-                        ---
-                        Many .condition.type values are consistent across resources like Available, but because arbitrary conditions can be
-                        useful (see .node.status.conditions), the ability to deconflict is important.
-                        The regex it matches is (dns1123SubdomainFmt/)?(qualifiedNameFmt)
+                      description: type of condition in CamelCase or in foo.example.com/CamelCase.
```

Comment:
Lots of deletions in CRD spec here which explains the large overall diff. In most cases, this is removing whitespace, but in some cases, this is removing some part of the description.